src/cli/repo-checkout.js

```diff
@@ -125,34 +125,47 @@ export default async function main(config, argv) {
     projects: {}
   };
 
-  await Promise.all(projects.map(async (project) => {
-    let projectStart = Date.now();
+  let archivesToExtract = await Promise.all(projects.map(async (project) => {
+    let downloadStart = Date.now();
+    stats.projects[project.name] = {duration: 0};
+
     let repoPath =
       fsPath.join(args.directory, '.repo', 'projects', `${project.path}.git`);
```

Comment:
repoPath will be the target directory after extraction.  We could do something smarter with building out the directories first and then extracting so perhaps we could do things in parallel but this is a simple fix now and works without adding a huge amount of time.

Since tc-vcs has never really worked right since it's inception and we are losing trust over it, I want to get some clear wins with it and we can make it smarter or rewrite it at some point (in Go)